### PR TITLE
Broadbandy AMP Integration 

### PR DESCRIPTION
**Pull request by BroadBandyDev**
Hi there,
We'd like to integrate our Ad-Server with AMP. We've contributed to AMP as per documentation. Please check & approve.
Thanks in advance!

Tijl Verbert

### CODE REVIEW

**Comment by BroadBandyDev** [comment]
HI, i see the we are still missing the approval, anything we can do from our side?

File: .gitignore [hunk 1]
```diff
@@ -43,3 +43,6 @@ npm-debug.log
 .firebase
 firebase
 firebase.json
+
+#Editor configs
+.idea
```
Comment (calebcordry) [inline .gitignore:48]: please remove. If you would like this added, please open a separate issue so that we can discuss with the infra folks.

**Review by calebcordry — COMMENTED** [review]
Thanks for contributing! just one nit.

File: 3p/integration.js [hunk 2]
```diff
@@ -374,6 +375,7 @@ register('blade', blade);
 register('bodymovinanimation', bodymovinanimation);
 register('brainy', brainy);
 register('bringhub', bringhub);
+register('broadbandy', broadbandy);
```
Comment (powerivq) [inline 3p/integration.js:378]: Please revert the changes in this file. It is not needed.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (BroadBandyDev) [reply .gitignore]: removed

File: 3p/integration.js [hunk 3]
```diff
@@ -90,6 +90,7 @@ import {bidtellect} from '#ads/vendors/bidtellect';
 import {blade} from '#ads/vendors/blade';
 import {brainy} from '#ads/vendors/brainy';
 import {bringhub} from '#ads/vendors/bringhub';
+import {broadbandy} from '#ads/vendors/broadbandy';
```
Comment (powerivq) [inline 3p/integration.js:93]: Remove the import as well.

**Review by powerivq — CHANGES_REQUESTED** [review]

File: 3p/integration.js (hunk 2 above)
Comment (BroadBandyDev) [reply 3p/integration.js]: hi we removed the requested line
Thanks

File: 3p/integration.js (hunk 3 above)
Comment (BroadBandyDev) [reply 3p/integration.js]: done, thanks

**Comment by powerivq** [comment]
Nit: https://app.circleci.com/pipelines/github/ampproject/amphtml/16425/workflows/b3e6554b-2b27-4f25-8a48-0284f3434ffb/jobs/293543 fix the check error.

**Review by powerivq — APPROVED** [review]
